python/tests/conftest.py

```diff
@@ -42,20 +48,16 @@
 BUCKET_ID = 0
 
 
+def run_server(port):
```

Comment:
Can we make it clear what kind of server this is?

Reply:
That is fair - I'll make it private too.